stdlib/public/SDK/Foundation/DateComponents.swift

```diff
@@ -339,9 +339,8 @@ extension DateComponents : _ObjectiveCBridgeable {
     }
 
     public static func _unconditionallyBridgeFromObjectiveC(_ source: NSDateComponents?) -> DateComponents {
-        var result: DateComponents?
-        _forceBridgeFromObjectiveC(source!, result: &result)
-        return result!
+        guard let src = source else { return DateComponents() }
+        return DateComponents(reference: src)
```

Comment:
most of those are internal; I think it would be interesting to perhaps make all bridgeable types have a public init method that takes a reference and a counterpart to the reference types that take the value (however that may seem too C++ to some...)